py/kubeflow/kfctl/testing/util/aws_util.py

```diff
@@ -0,0 +1,14 @@
+import os
+import logging
+from kubeflow.testing import util
+from kubeflow.testing.cloudprovider.aws import util as aws_util
+
+
+def aws_auth_load_kubeconfig(cluster_name):
+    if os.getenv("AWS_ACCESS_KEY_ID") and os.getenv("AWS_SECRET_ACCESS_KEY"):
```

Comment:
minor: logs seems not helpful here. `aws eks` command will resolve auth chain and you will find envs from pod spec directly. (Argo supports this by default)

Reply:
Yeah make sense, let me reword